stage1/init/init.go

```diff
@@ -296,6 +296,13 @@ func getArgsEnv(p *stage1commontypes.Pod, flavor string, debug bool, n *networki
 		nsargs := stage1initcommon.VolumesToKvmDiskArgs(p.Manifest.Volumes)
 		args = append(args, nsargs...)
 
+		// set hostname inside pod
```

Comment:
if not a problem, please add a comment referring to [systemd-hostname](https://www.freedesktop.org/software/systemd/man/hostname.html) why writing /etc/hostname is enough